Source/Scene/Cesium3DTileset.js

```diff
@@ -879,6 +879,19 @@ import TileOrientedBoundingBox from './TileOrientedBoundingBox.js';
     }
 
     defineProperties(Cesium3DTileset.prototype, {
+
+        /**
+         * NOTE: This getter exists so that `Picking.js` can differentiate between
+         *       PrimitiveCollections and Cesium3DTileset objects without inflating
+         *       the size of the module via `instance of Cesium3DTileset`
+         */
+
+        isCesium3DTileset: {
+            get: function() {
+                return true;
+            }
+        }
```

Comment:
This is why CI is failing. You can run `npm run eslint` locally which will also catch this.
```suggestion
        },
```